api_v3/services/BaseEntryService.php

```diff
@@ -896,4 +896,49 @@ public function cloneAction( $entryId, $cloneOptions=null)
 		$clonedEntry = myEntryUtils::copyEntry($coreEntry, $this->getPartner(), $coreClonedOptionsArray);
 		return $this->getEntry($clonedEntry->getId());
 	}
+
+	/**
+	 * This action delivers all data relevant for player
+	 * @action getPlayingData
+	 * @param string $entryId
+	 * @param KalturaEntryContextDataParams $contextDataParams
+	 * @return KalturaPlayingResult
+	 * @throws KalturaErrors::ENTRY_ID_NOT_FOUND
+	 */
+	function getPlayingDataAction($entryId, KalturaEntryContextDataParams $contextDataParams)
```

Comment:
Done forget to add this function to globalPartnerAllowed & kalturaNetworkAllowed